scripts/src/public/services/orders.ts

```diff
@@ -111,7 +118,37 @@ export async function createMarketplaceOrder(offerId: string, userId: string, lo
 		throw new Error(`offer ${ offerId } cap reached`);
 	}
 
-	logger.info("created new open order", { offerId, userId, orderId: order.id });
+	logger.info("created new open marketplace order", { offerId, userId, orderId: order.id });
+
+	return {
+		id: order.id,
+		expiration_date: order.expirationDate!.toISOString(),
+	};
+}
+
+export async function createExternalOrder(jwt: string, userId: string, logger: LoggerInstance): Promise<OpenOrder> {
+	const offer = await validateSpendJWT(jwt, logger);
+	let order = await db.ExternalOrder.findOne({
+		where: {
+			userId,
+			status: "opened",
+			offerId: offer.id
+		}
+	});
+
+	if (!order) {
+		order = db.ExternalOrder.new({
```

Comment:
aren't you storing the `offer.title`, `offer.description` under `order.meta`?

Reply:
2